src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ListingServiceTest.java

```diff
@@ -43,7 +43,6 @@
 import uk.gov.hmcts.ecm.common.model.reports.respondentsreport.RespondentsReportCaseData;
```

Comment:
May be a unit test for the case when CauseListDate(s) are empty.

Reply:
Write a unit test for dates 01 January and 01 December. Which will come first?

Reply:
@gauravchawla73463 - 2 tests created, 1 for listed date and other for causeListDate, both will cause exceptions
@jthmcts - added extra code for the comparator and a unit test to test 01 December and Jan